dashboard/app/assets/stylesheets/application.scss

```diff
@@ -422,6 +422,16 @@ img.video_thumbnail {
   }
 }
 
+/* When we are using a dark theme in Google Blockly, we want the main Blockly area (inside #codeWorkspace)
+ * to have a 1px separation between it and the PaneHeader above it.
+*  */
+#codeWorkspace > .cdomoderndark-theme {
+  margin-top: 1px;
```

Comment:
The separation looks really nice in modern theme. 

![Screen Shot 2023-05-19 at 12 01 04 PM](https://github.com/code-dot-org/code-dot-org/assets/107423305/0233a322-b788-479f-a296-101f8a4a3eab)

But the line seems to disappear when dark mode is combined with other themes (high contrast, protanopia). I checked because high contrast changes the sizes of elements, but I was surprised to see the line disappear with the color vision deficiency themes.

High contrast dark:

![Screen Shot 2023-05-19 at 12 04 26 PM](https://github.com/code-dot-org/code-dot-org/assets/107423305/cac1eb14-0c82-4edb-b8ab-2f7d1207eea5)




Protanopia dark:

![Screen Shot 2023-05-19 at 12 00 16 PM](https://github.com/code-dot-org/code-dot-org/assets/107423305/caf14d96-179a-4606-a0ec-ab63b0744c5b)

Reply:
Nice catch!  I just pushed an update which should make it work in all of the dark modes.